packages/scene-composer/src/components/panels/scene-components/tag-style/ColorPicker/ColorPickerUtils/SvgParserHelper.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+export const replaceFillAttribute = (element: Element, selectedColor: string): void => {
+  const tagName = element.tagName.toLowerCase();
+
+  if (tagName === 'ellipse') {
+    if (element === undefined || selectedColor === undefined) {
```

Comment:
Move this to outer as it's the same in both cases

Reply:
next cr.